documentation/faq/FAQ.adoc

```diff
@@ -100,4 +100,117 @@ Note that whitelisting only applies while downloading resources (jars and jnlps)
 Thus, an application can open a connection to a server which is not in the white list.
 
 It is also possible to specify the content of the whitelist when installing OWS (unattended installation),
-see https://openwebstart.com/installation/ and https://openwebstart.com/configuration/ for further details.
+See https://openwebstart.com/installation/ and https://openwebstart.com/configuration/ for further details.
+
+==== How to run OpenJFX based JavaFX applications with OpenWebStart?
+
+*With JDK 8*
+
+To be able to run a JavaFX application with OWS using Java 8 requires an installation of Java 8 JVM that includes JavaFX.
+Some of the vendors that have JavaFX as part of their Java 8 JVMs are Oracle, Azul, BellSoft and Amazon. OpenJDK 8 from
+Adopt does not include JavaFX.
+
+Using the JVM Server feature of OWS JVM Manager it can be ensured that a suitable JDK 8 with JavaFX will be installed on
+the machine for OWS to start a JavaFX app. The required JVM from a preferred vendor can be specified in the Jnlp file:
+
+....
+<?xml version="1.0" encoding="utf-8"?>
+<jnlp spec="1.0+" codebase="$$codebase">
```

Comment:
use of $$ variable might be confusing here, maybe use some fake url instead?

Reply:
This is just a sample. Users knows what to put in codebase.